sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/jdbc/JDBCTableCatalog.scala

```diff
@@ -278,7 +278,9 @@ class JDBCTableCatalog extends TableCatalog with SupportsNamespaces with Logging
     }
   }
 
-  override def dropNamespace(namespace: Array[String]): Boolean = namespace match {
+  override def dropNamespace(
+      namespace: Array[String],
+      cascade: Boolean): Boolean = namespace match {
```

Comment:
@beliefer do you know how JDBC support DROP DATABASE CASCADE?

Reply:
`DROP DATABASE` fails if exists table in database.
` DROP DATABASE CASCADE` drops all the table in database and database itself.

Reply:
Can you create a followup PR to fix JDBC source? We need to respect the cascade parameter. thanks!

Reply:
@cloud-fan Thank you for the ping.